### PR TITLE
LPS-120579 Fix author avatar alignment

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:workflow

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:workflow

File: modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -309,6 +309,14 @@
...
 		font-weight: bold;
 		margin-right: 10px;
+
+		img {
+			vertical-align: baseline;
+		}
+
+		svg {
+			vertical-align: baseline;
+		}
```
Comment (albinojunior) [inline modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/css/main.scss:319]: @naraandrade you could use `img, svg { ... }` to apply the same style to one more selector.

Comment (naraandrade) [reply modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/css/main.scss:319]: > @naraandrade you could use `img, svg { ... }` to apply the same style to one more selector.

Thanks, @albinojunior :) I'll do that

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:workflow

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:workflow

**Comment by naraandrade** [comment]
ci:test:workflow

**Comment by naraandrade** [comment]
ci:test:workflow

**Review by albinojunior — APPROVED** [review]

**Comment by albinojunior** [comment]
ci:forward

**Pull request closed without merging**